Code/Tools/ProjectManager/Source/ProjectButtonWidget.h

```diff
@@ -71,7 +75,13 @@ namespace O3DE::ProjectManager
         QLabel* m_warningIcon = nullptr;
         QSpacerItem* m_warningSpacer = nullptr;
 
+        QLabel* m_cloudIcon = nullptr;
+
         QLabel* m_buildingAnimation = nullptr;
+        QLabel* m_downloadMessageLabel = nullptr;
+
+        QProgressBar* m_progessBar = nullptr;
+        QLabel* m_progressMessageLabel = nullptr;
```

Comment:
any reason why these need to be pointers I think this could work just as well as member variables and access through a reference?